xoinvader/tests/test_background.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+
+from xoinvader.background import Background, Chunk, load_chunks
+from xoinvader.common import Settings
+from xoinvader.utils import Point, Surface
+
+from copy import copy
+
+def test_chunk():
+    c = Chunk("test")
+    assert not c.lines
+    assert not c.length
+    assert c.name == "test"
+
+    c.add_line("wut")
+    assert c.lines[0] == "wut"
+    assert c.length == 1
+
+    c[0] = "ugh"
+    assert c[0] == "ugh"
+
```

Comment:
two empty lines separates finctions and class definitions in module's scope, one empty line between methods in class.